internal/cmd/connect/command_plugin_install.go

```diff
@@ -408,7 +435,7 @@ func unzipPlugin(pluginManifest *manifest, zipFiles []*zip.File, pluginDir strin
 	return nil
 }
 
-func checkLicenseAcceptance(pluginManifest *manifest, force bool) error {
+func checkLicenseAcceptance(pluginManifest *manifest, prompt *form.RealPrompt, force bool) error {
```

Comment:
I think RealPrompt implements Prompt? It seems like the intention is to use Prompt here, so we can pass in a mock prompt during unit tests (which you should write)